src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -148,6 +148,10 @@
       + "Elasticsearch rejects due to some malformation of the document itself, such as an index"
       + " mapping conflict or a field name containing illegal characters. Valid options are "
       + "'ignore', 'warn', and 'fail'.";
+  public static final String USE_UPDATE_CONFIG = "use.update";
+  private static final String USE_UPDATE_DOC = "Whether to use UPDATE (UPSERT) instead of "
+      + "INSERT method. This allows documents to be updated rather than overwritten "
+      + "(e.g. new fields added). Valid options are 'true', 'false'. Default to 'false'.";
```

Comment:
Actually, I do not see any other ways of update. But that is only my personal understanding at the moment :). I am OK with that. Since we are dealing with writing, I suggest 'write.method'. I found word update missleading.

Reply:
@rhauch, I have added `write.method` string config. Allowed values are `insert`, `update`, `upsert`. Let me know if that's OK with you.